src/gatling/simulations/scenarios/Solicitor_Bails.scala

```diff
@@ -0,0 +1,1383 @@
+    .group("XUI_Bails_500_Upload_Bail") {
+
+      exec(http("XUI_Bails_500_005_Upload_Bail")
+        .post("/documentsv2")
+        .headers(Headers.commonHeader)
+        .header("accept", "application/json, text/plain, */*")
+        .header("content-type", "multipart/form-data")
+        .header("X-XSRF-TOKEN", "${XSRFToken}")
+        .formParam("classification", "PUBLIC")
+        .formParam("caseTypeId", "Bail")
+        .formParam("jurisdictionId", "IA")
+        .bodyPart(RawFileBodyPart("files", "120KB.pdf")
+          .fileName("120KB.pdf")
+          .transferEncoding("binary"))
+        .asMultipartForm
+        .check(jsonPath("$.documents[0].hashToken").saveAs("DocumentHash"))
+        .check(jsonPath("$.documents[0]._links.self.href").saveAs("DocumentURL")))
+
+      .exec(http("XUI_Bails_500_010_Upload_Bail")
+        .post("/data/case-types/Bail/validate?pageId=uploadBailSummarybailSummaryDocumentUpload")
+        .headers(Headers.commonHeader)
+        .header("accept", "application/vnd.uk.gov.hmcts.ccd-data-store-api.case-data-validate.v2+json;charset=UTF-8")
+        .header("x-xsrf-token", "${XSRFToken}")
+        .body(ElFileBody("bodies/bails/BailsBailUpload.json"))
+        .check(substring("uploadBailSummaryDocs")))
+
+      .exec(Common.healthcheck("%%2Fcases%2Fcase-details%2F${case_Id}%2Ftrigger%2FuploadBailSummary%2Fsubmit"))
+
+    }
+    .pause(MinThinkTime, MaxThinkTime)
+
+
+    /*======================================================================================
+    * Bail Summary Submit
+    ======================================================================================*/
+
+    .group("XUI_Bails_510_Upload_Bail_Submit") {
+
+      exec(http("XUI_Bails_510_005_Upload_Bail_Submit")
+        .post(BaseURL + "/data/cases/${case_Id}/events")
+        .headers(Headers.commonHeader)
+        .header("accept", "application/vnd.uk.gov.hmcts.ccd-data-store-api.create-event.v2+json;charset=UTF-8")
+        .header("x-xsrf-token", "${XSRFToken}")
+        .body(ElFileBody("bodies/bails/BailsUploadSubmit.json"))
+        .check(jsonPath("$.state").is("bailSummaryUploaded")))
+
+      .exec(Common.healthcheck("%2Fcases%2Fcase-details%2F${case_Id}%2Ftrigger%2FuploadBailSummary%2FuploadBailSummarybailSummaryDocumentUpload"))
+
+
+    }
+    .pause(MinThinkTime, MaxThinkTime)
+
+
+  val RecordBailDecision =
+
+    /*======================================================================================
+    * Load case
+    ======================================================================================*/
+
+    group("XUI_Bails_520_Open_Case") {
+
+      exec(Common.healthcheck("%2Fcases%2Fcase-details%2F${case_Id}"))
+
+      .exec(http("XUI_Bails_520_005_Open_Case")
+        .get("/data/internal/cases/${case_Id}")
+        .headers(Headers.commonHeader)
+        .header("accept", "application/vnd.uk.gov.hmcts.ccd-data-store-api.ui-case-view.v2+json")
+        .check(jsonPath("$.case_id").is("${case_Id}")))
```

Comment:
maybe check the $.state field if useful

Reply:
Changed it to check the state.name is 'Bail summary'